features/models.py

```diff
@@ -241,6 +241,37 @@ def __str__(self):
         return f"{self.tag}"
 
 
+class FeatureTeaser(TranslatableModel):
+    feature = models.OneToOneField(
+        Feature,
+        on_delete=models.CASCADE,
+        related_name="teaser",
+        verbose_name=_("teaser"),
+    )
+    translations = TranslatedFields(
+        header=models.CharField(
+            max_length=64,
+            blank=True,
+            verbose_name=_("header"),
+            help_text=_("An opening, e.g. Starting' from 'Starting from 7€/day.'"),
+        ),
+        main=models.CharField(
```

Comment:
Just wondering if `content` would be a more suitable name for this field. :thinking: Well, your call.